lib/OpenGuides/Config.pm

```diff
@@ -151,7 +152,14 @@ sub _init {
             warn "Don't know what to do with variable '$var'";
         }
     }
-
+    # Override config from file or defaults if an environment variable
+    # is set for a variable. e.g OPENGUIDES_CONFIG_read_only
+    foreach my $var ( @variables ) {
+       my $envvar = "OPENGUIDES_CONFIG_" . uc ($var);
+       if (exists $ENV{$envvar}) {
+           $self->$var ("$ENV{$envvar}");
```

Comment:
Environment variables are always strings, so no need to quote it here.

Reply:
okay.

Reply:
updated.